src/main/java/com/greenfox/notification/service/EmailSenderService.java

```diff
@@ -0,0 +1,37 @@
+package com.greenfox.notification.service;
+
+import com.greenfox.notification.model.classes.Data;
+import com.sendgrid.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Service
+public class EmailSenderService {
```

Comment:
Separate this service to:
 - Something that push it to the queue
 - Something that consume it from the queue
 - Something that sends an email